package.json

```diff
@@ -17,7 +17,12 @@
   },
   "devDependencies": {
     "asmcrypto.js": "0.0.11",
+    "babel-core": "^6.25.0",
+    "babel-loader": "^7.1.1",
+    "babel-plugin-add-module-exports": "^0.2.1",
     "babel-polyfill": "^6.23.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
     "browserify": "^14.4.0",
```

Comment:
we should follow up to remove this, but i didn't want to mess with `test--browser` yet